Add more tools using `gh` utility to access github API

blag opened the pull request:
I've been working on these tools, since I wasn't pleased with the existing tools.

These tools are buggy, unfinished, and poorly documented, but since they might be useful to other people I'm publishing them in this draft PR.

blag committed:
Add a few more half-finished tools

arm4b commented on rerun-workflows.sh:
```diff
@@ -0,0 +1,131 @@
+#!/bin/bash
+#
+# A helper script to bootstrap a new StackStorm Exchange pack.
```
:+1:

arm4b reviewed (commented):
Yeah, would be indeed nice including detailed description at the header for other files too.
This will help someone better navigating them.

cognifloyd committed:
Use github cli (gh) in helper functions

cognifloyd committed:
add _gh_repos_tags function

cognifloyd committed:
Merge pull request #3 from cognifloyd/add-more-tools

Use github cli (gh) in helper functions

cognifloyd committed:
add script descriptions

cognifloyd committed:
drop CircleCI specific workflow

we are replacing CircleCI with GHA. Drop it.
It can be retrieved from git history if needed.

cognifloyd commented:
This is required for https://github.com/StackStorm/community/issues/63

cognifloyd commented:
I'm merging this. We can add more docs or enhancements in follow-up PRs.
